docs/guide-platform-support.md

```diff
@@ -3,41 +3,44 @@ id: guide-platform-support
 title: Supported Environments
 ---
 
-## Node.js Support
+## Node-API (N-API)
+
+Since v10 [`@serialport/bindings-cpp`](api-bindings-cpp.md) has used [Node-API](https://nodejs.org/api/n-api.html) (also known as `N-API`) to allow easy use with any recent NodeJS and Electron version. (And in theory any V8 based environment.) We target Node-API version 6 as it covers the current NodeJS and electron support requirements.
+
+### Node.js Support
 
 SerialPort officially supports all Active, Maintenance and Current LTS releases of Node.js. For information on what versions this currently covers see the [Node.js Release Schedule](https://github.com/nodejs/Release#release-schedule).
 
-## Electron Support
+### Electron Support
 
-SerialPort officially supports the last 3 versions of Electron per it's [8 week release cadence](https://www.electronjs.org/blog/8-week-cadence). We usually lag a few weeks behind releases while build tools are updated.
+SerialPort officially supports the last 3 versions of Electron per it's [8 week release cadence](https://www.electronjs.org/blog/8-week-cadence). We sometimes lag a few weeks behind releases while build tools are updated.
 
 ## Supported Platforms and Architectures
 
-The platforms, architectures that `serialport` supports are the following;
+The [`@serialport/bindings-cpp`](api-bindings-cpp.md) package uses [`prebuildify-cross`](https://www.npmjs.com/package/prebuildify-cross) to build binaries for a [number of platforms and C libraries](https://github.com/prebuild/docker-images#images). We are able to run a CI for only the major platforms offered by GitHub Actions. Those platforms are fully supported while the others are a best effort.
 
-| Platform / Arch | Supported |
-|       ---       | --- |
-| Linux / ia32¹   |  ☐  |
-| Linux / x64     |  ☑  |
-| Linux / ARM v6⁴ |  ☐  |
-| Linux / ARM v7⁴ |  ☐  |
-| Linux / ARM v8⁴ |  ☐  |
-| Linux / MIPSel⁴ |  ☐  |
-| Linux / PPC64⁴  |  ☐  |
-| OSX / x64³      |  ☑  |
-| Windows² / x86  |  ☑  |
-| Windows² / x64  |  ☑  |
+The platforms, architectures that `serialport` supports are the following;
 
+| Platform / Arch | Supported | Built |
+|       ---       | --- | --- |
+| Linux / ia32¹   |  ☐  |  ☐  |
+| Linux / x64     |  ☑  |  ☑  |
+| Linux / ARM v6  |  ☐  |  ☑  |
+| Linux / ARM v7  |  ☐  |  ☑  |
+| Linux / ARM64   |  ☐  |  ☑  |
+| OSX / x64³      |  ☑  |  ☑  |
+| OSX M1³         |  ☑  |  ☑  |
+| Windows² / x86  |  ☑  |  ☑  |
```

Comment:
is windows x86 still supported.  perhaps I'm mixing this up with the dropped Linux support, but I thought windows was 64bit only now too...